app/models/criteria_review.rb

```diff
@@ -0,0 +1,16 @@
+class CriteriaReview < ApplicationRecord
+  include WithScoring
+
+  self.table_name = 'criteria_reviews'
```

Comment:
why do we need this?

Reply:
Remnant from an earlier iteration. No longer needed.